src/test_helpers.rs

```diff
@@ -0,0 +1,46 @@
+use crate::config::DataContext;
+use std::path::PathBuf;
+use uuid::Uuid;
+
+pub fn create_test_context() -> DataContext {
+    let test_id = Uuid::new_v4().to_string();
+    let base_dir = PathBuf::from(format!("test_data_{}", test_id));
+    DataContext::new(base_dir)
+}
+
+pub fn cleanup_test_context(context: &DataContext) {
+    let _ = std::fs::remove_dir_all(&context.base_dir);
```

Comment:
ここも不要な代入せず ok() しちゃってよさそうです

```suggestion
    std::fs::remove_dir_all(&context.base_dir).ok();
```